### PR TITLE
changed behaviour of the non CDM data sources

### PR DESCRIPTION
**Pull request by johnSamilin**

### CODE REVIEW

File: src/modules/CdmSourceList/components/ViewEdit/Banner/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+  *
+  * Copyright 2017 Observational Health Data Sciences and Informatics
```
Comment (agackovka) [inline src/modules/CdmSourceList/components/ViewEdit/Banner/index.js]: 2018

File: src/modules/CdmSourceList/components/ViewEdit/Banner/style.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/*
+  *
+  * Copyright 2017 Observational Health Data Sciences and Informatics
```
Comment (agackovka) [inline src/modules/CdmSourceList/components/ViewEdit/Banner/style.scss]: 2018

File: src/modules/CdmSourceList/components/ViewEdit/presenter.jsx [hunk 3]
```diff
@@ -40,15 +40,13 @@ function ViewEdit({ isLoading, dataSourceName, published, modelType }) {
...
         <Toolbar />
         <div {...classes({ element: 'content', modifiers: { empty: !reportsAvailable } })}>
-          <div{...classes({ element: 'row', modifiers: { empty: !reportsAvailable }, extra: 'row' })}>
-            {reportsAvailable
-              ? <div className="col-xs-12 col-md-6">
-                  <AchillesSettings />
-                </div>
-              : <EmptyState message={'Generating reports is unavailable for non-CDM data sources'} />
-            }
+          <div {...classes({ element: 'row', modifiers: { empty: !reportsAvailable }, extra: 'row' })}>
```
Comment (pavgra) [inline src/modules/CdmSourceList/components/ViewEdit/presenter.jsx]: Why the "empty" modificator is left?

File: src/modules/CdmSourceList/components/ViewEdit/Banner/style.scss [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  * Authors: Pavel Grafkin, Alexander Saltykov, Vitaly Koulakov, Anton Gackovka, Alexandr Ryabokon, Mikhail Mironov
+  * Created: Friday, June 8, 2018 2:07 PM
+  *
+  */
+
+@import 'styles/vars-and-mixins.scss';
+ 
+.#{$namespace} { 
+  &banner {
+    background: $grey-dark;
```
Comment (pavgra) [inline src/modules/CdmSourceList/components/ViewEdit/Banner/style.scss]: We discussed system settings alike color

**Pull request merged**